docs/quickstart.html

```diff
@@ -158,15 +158,78 @@ <h4 class="anchor-heading">
         </h4>
 
         <p>
-            You probably have lots of data in existing systems like relational databases or traditional messaging systems,
-            along with many applications that already use these systems.
-            <a href="/documentation/#connect">Kafka Connect</a> allows you to continuously ingest
-            data from external systems into Kafka, and vice versa.  It is thus very easy to integrate existing systems with
-            Kafka. To make this process even easier, there are hundreds of such connectors readily available.
+            Reading data from the console and writing it back to the console is a convenient place to start, but you'll probably want
```

Comment:
Should we keep the existing paragraph and add this after it? I think it helps introduce Connect.

Reply:
I think the existing paragraph and the new ones cover similar things so I've had a go at combining them. Let me know if you think it reads ok